libs/zephyr-rspress-plugin/src/withZephyrRspressSSG.ts

```diff
@@ -0,0 +1,31 @@
+import { xpack_zephyr_agent } from 'zephyr-xpack-internal';
+import type { ZephyrRspressPluginOptions } from './types';
+import { buildAssetMapFromFiles } from './internal/assets/buildAssets';
+import { buildStats } from './internal/stats/buildStats';
+
+export async function withZephyrRspressSSG({
```

Comment:
Can we rename this in a more meaningful way?
we have zephyrRspressSSGPlugin in here and then this function with similar naming.

rename it in a way that it tells better what it is doing, please